source/support.tex

```diff
@@ -4395,9 +4395,9 @@
 \pnum
 The \tcode{weak_ordering} type is typically used
 as the result type of a three-way comparison operator\iref{expr.spaceship}
-that (a) admits all of the six
-two-way comparison operators\iref{expr.rel,expr.eq},
-and (b) does not imply substitutability.
+for a type that admits
+all of the six two-way comparison operators\iref{expr.rel,expr.eq}
+and for which equality need not imply substitutability,
```

Comment:
```suggestion
and for which equality need not imply substitutability.
```

Reply:
fixed

Reply:
(You need to resolve these comments; only PR authors and folks with commit authority can resolve. Yes, that's right, Github doesn't allow comment authors to resolve their own comments.)

Reply:
Yes, but it seems more friendly to the comment author to have him review the unresolved comment back-and-forth before hiding it from view.

Reply:
That does make sense given that a great many of our commentors are not full-time GitHub residents.